src/GovUk.Education.ExploreEducationStatistics.Admin/Mappings/MappingProfiles.cs

```diff
@@ -219,13 +219,19 @@ public MappingProfiles()
 
             CreateMap<ContentSection, ContentSectionViewModel>()
                 .ForMember(dest => dest.Content,
-                m => m.MapFrom(section => section.Content.OrderBy(contentBlock => contentBlock.Order)));
+                m => m.MapFrom(section =>
+                    section.Content.OrderBy(contentBlock => contentBlock.Order)));
 
             CreateMap<MethodologyVersion, ManageMethodologyContentViewModel>()
                 .ForMember(dest => dest.Content,
-                    m => m.MapFrom(methodology => methodology.Content.OrderBy(contentSection => contentSection.Order)))
+                    m => m.MapFrom(methodologyVersion =>
+                        methodologyVersion.Content.OrderBy(contentSection => contentSection.Order)))
                 .ForMember(dest => dest.Annexes,
-                    m => m.MapFrom(methodology => methodology.Annexes.OrderBy(annexSection => annexSection.Order)));
+                    m => m.MapFrom(methodologyVersion =>
+                        methodologyVersion.Annexes.OrderBy(annexSection => annexSection.Order)))
+                .ForMember(dest => dest.Notes,
+                    m => m.MapFrom(methodologyVersion =>
+                        methodologyVersion.Notes.OrderByDescending(note => note.DisplayDate)));
```

Comment:
Do we always have a DisplayDate?

Reply:
Yes, `DisplayDate` can't be null at the moment. I put in a change this morning to require it in the add and update request models when managing notes. If a request did ever hit the service layer without including a display date we default it to todays date.